token-lending/program/src/processor.rs

```diff
@@ -1339,6 +1339,14 @@ fn process_borrow_obligation_liquidity(
         msg!("Borrow reserve is stale and must be refreshed in the current slot");
         return Err(LendingError::ReserveStale.into());
     }
+    if Decimal::from(liquidity_amount)
```

Comment:
We should add a case for when `borrow_limit == u64::MAX` that just passes through. This is more of a nit since the token program stores amounts as `u64` anyway, but it's good practice to have.

Reply:
we talked about this offline, same reason we didn't (the token program u64 cap)

Reply:
I see. We'd save a tiny bit of computation right?

Guess it's not too important either way.

Reply:
saving computation sometimes doesn't really matter right? worst case is all the matters so better to keep consistent compute so we dont accidentally introduce bugs that aren't caught cause edge case overcompute